js/packages/teams-dialogs/src/prompts/teamsBotSsoPrompt.ts

```diff
@@ -0,0 +1,470 @@
+/**
```

Comment:
Use same format as other tests: `____.spec.ts`?

Reply:
I am a little confused about the naming. In the pr you mentioned above, I found tests are naming like `___.spec.ts` in `teams-ai`, but under `teams-dialogs`, tests are naming like `____.test.js`. Which one should I follow?

Reply:
Oh, interesting point... I'm assuming `teams-dialogs` are the tests directly ported from `botbuilder-dialogs`. It should be fine to keep `__.tests.js`, though I'm not sure the purpose of using js.

Thank you for pointing this out!